dali/pipeline/workspace/workspace.h

```diff
@@ -50,25 +51,25 @@ class ArgumentWorkspace {
     argument_inputs_.clear();
   }
 
-  void AddArgumentInput(shared_ptr<TensorList<CPUBackend>> input, const std::string &arg_name) {
+  void AddArgumentInput(shared_ptr<TensorVector<CPUBackend>> input, const std::string &arg_name) {
     argument_inputs_[arg_name] = std::move(input);
   }
 
-  void SetArgumentInput(shared_ptr<TensorList<CPUBackend>> input, const std::string &arg_name) {
+  void SetArgumentInput(shared_ptr<TensorVector<CPUBackend>> input, const std::string &arg_name) {
```

Comment:
This function is never used - and the semantics are quite pointless. Please remove it.

Reply:
done